static/bidder-params/interactiveOffers.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Interactive Offers Adapter Params",
+  "description": "A schema which validates params accepted by Interactive Offers adapter",
+  "type": "object",
+  "properties": {
+    "pubid": {
```

Comment:
The publisher id is already available in the OpenRTB request as either app.publisher.id or site.publisher.id. Why does your adapter need its own?

Reply:
the publisher.id under the OpenRTB standards is not a required value, for us it is